cmd/admin-scanner-status.go

```diff
@@ -305,37 +305,43 @@ func (m *scannerMetricsUI) View() string {
 
 	title := metricsTitle
 	ui := metricsUint64
-	const wantCycles = 16
 	addRow("")
-	if len(sc.CyclesCompletedAt) < 2 {
-		addRow("Last full scan time:             Unknown (not enough data)")
+
+	if sc.CurrentCycle == 0 && sc.CurrentStarted.IsZero() && sc.CyclesCompletedAt == nil {
+		addRowF("     "+title("Scanning:")+" %d bucket(s)", sc.OngoingBuckets)
```

Comment:
what is the new metrics for seeing what is being scanner per bucket?

Reply:
yes, I am adding it to this PR